1.0.1/jessie/product/README.md

```diff
@@ -17,3 +17,25 @@ This image contains:
 ## Supported tags
 
 - [`1.0.1`, `latest` (*Dockerfile*)](https://github.com/aspnet/aspnet-docker/blob/master/1.0.1/jessie/product/Dockerfile)
+
+## Example Usage
+
+1. Create a Dockerfile for your application, the following example assumes you have already compiled your application (which is the expected use case for this image)
+
+  ```
+  FROM microsoft/aspnetcore
+  WORKDIR /app
+  ADD /app
```

Comment:
I've not seen ADD used with only one arg.  Did you intend for a '.' someplace?

Aside from that, I would suggest using COPY vs ADD per the Dockerfile [best practices](https://docs.docker.com/engine/userguide/eng-image/dockerfile_best-practices/#/add-or-copy).